webapp/public/locales/fr/translation.json

```diff
@@ -296,5 +296,9 @@
     "Thank you for responding to the": "Merci d’avoir soumis votre demande",
     "Note: You may need to change the file name to open the file on certain operating systems": "Remarque: vous devrez peut-être changer le nom du fichier pour ouvrir le fichier sur certains systèmes d'exploitation",
     "Next": "Continuer",
-    "Previous": "Retourner"
+    "Previous": "Retourner",
+    "User Title": "Titre de l'utilisateur",
```

Comment:
It doesn't look like you're using "User Title" and "User Id" any more - please remove